scripts/atomizer-loop.py

```
@@ -0,0 +1,111 @@
+            except Exception as e:
+                print(str(e))
+                attempts += 1
+    raise Exception("Couldn't reconnect to Mysql DB.")
+
+def get_unatomized_repos(connection):
+    cursor = connection.cursor(dictionary=True)
+    cursor.execute(f"""SELECT * FROM repos 
+                       WHERE 'Rust' IN (SELECT name 
+                                      FROM codes JOIN languages ON codes.language_id = languages.id
+                                      WHERE codes.repo_id = repos.id)
+                       AND repos.status_id = 1""")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)